driver/pelican/pelican.go

```diff
@@ -169,6 +190,49 @@ func (pel *Pelican) GetStatus() (*PelicanStatus, error) {
 		}
 	}
 
+	// Thermostat History Object Request to retrieve time stamps from past hour
+	timezone, timeErr := time.LoadLocation(pel.timezone)
+	if timeErr != nil {
+		return nil, fmt.Errorf("Invalid Timezone specified in pelican struct: %v\n", timeErr)
+	}
+	diff := time.Now().Add(-1 * time.Hour)
+	endTime := time.Now().In(timezone).Format(time.RFC3339)
+	startTime := diff.In(timezone).Format(time.RFC3339)
```

Comment:
This is still confusing because `diff` isn't a time difference, as its name would imply. I say just make a current `time.Time` instance from `time.Now` as a variable, then create `endTime` and `startTime` from it directly.